python/lsst/ts/cRIOpy/GUI/ApplicationControlWidget.py

```diff
@@ -38,6 +39,59 @@
 from lsst.ts.idl.enums.MTM1M3 import DetailedState
 
 
+class HPWarnings:
+    def __init__(self, m1m3):
+        self.faultHigh = self.faultLow = None
+        self.warningHigh = self.warningLow = None
+
+        self._faultLow = self._faultLowRaising = None
+
+        m1m3.hardpointActuatorSettings.connect(self.hardpointActuatorSettings)
```

Comment:
I suspect this might introduce the cyclic dependency?

```
m1m3.hardpointActuatorSettings.connect(self.hardpointActuatorSettings)
```

Anyway, introduce this dependency in the instantiation is weird to me.

Reply:
Did not cause cyclic dependency - why? Anway, was moved into main widget, as state needs to be set (if detailedState event was received before hardpoint..Settings).